### DISCUSSION TITLE
Take processed policy violations into account when adding new project version

### DISCUSSION THREAD

**Discussion by software-testing-professional** [DependencyTrack/dependency-track#1597] [Q&A]
We use Dependency-Track for open source license clearing. 
A configured license whitelist contains a bunch of open source licenses, which are ok in our use case. 
And a policy (`license group IS NOT whitelist`) raises a policy violation issue, whenever other licenses are detected. 
Once all license issues are processed, a project is "cleared". 

When a new version of a project is created (manually via `clone` or automatically via `uploadBOM`), the "whitelist" policy is applied again. 
And all policy violations show up again - although they have been processed in a previous version of the project. 
This results in huge manual effort.

My question is now, if there is any way to take processed violations into account when I create a new project version. 
So that a policy violation, that was suppressed in v1.0.0 of my project remains suppressed in v1.0.1). 

The only approach I currently see is to script this behavior via REST API. 

Thank you in advance for your help!

Best regards, Michael

**Comment by stevespringett** [comment]
When cloning a project, there's an option to include audit decisions. No such option currently exist for policy decisions. I would recommend creating an enhancement request so the team can track it. PRs are also welcome.
